translator/internaldata/oc_to_resource.go

```diff
@@ -80,7 +80,7 @@ func ocNodeResourceToInternal(ocNode *occommon.Node, ocResource *ocresource.Reso
 	}
 
 	attrs := dest.Attributes()
-	attrs.InitEmptyWithCapacity(maxTotalAttrCount)
+	attrs.EnsureCapacity(maxTotalAttrCount)
```

Comment:
Same here.

Reply:
Not a problem, the output param is empty.